src/main/java/com/jvm_bloggers/frontend/public_area/contributors/ContributorsPage.java

```diff
@@ -33,17 +32,7 @@ protected void populateItem(ListItem<Contributor> item) {
 
         add(new ListView<Contributor>(
             SECOND_LEVEL_CONTRIBUTORS_LIST_ID,
-            contributors.slice(6, 26).toJavaList()
-        ) {
-            @Override
-            protected void populateItem(ListItem<Contributor> item) {
-                item.add(new ContributorDetails("contributorDetails", item.getModel()));
-            }
-        });
-
-        add(new ListView<Contributor>(
-            THIRD_LEVEL_CONTRIBUTORS_LIST_ID,
-            contributors.drop(26).toJavaList()
+            contributors.slice(6, 30).toJavaList()
```

Comment:
It would be good to limit this at call to github API. This implementation would fetch every contributor from github (multiple calls as we need to handle paging) and then filter them on our side. Instead it would be nice to fetch just the required number of contributors.